backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -39,16 +39,19 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
 
   def backoff: Backoff
 
+  def retryable: Boolean
+
   /**
     * If the `work` `Future` completes successfully, perform the `onSuccess` work, otherwise schedule
     * the execution of the `onFailure` work using an exponential backoff.
     */
   def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
     work onComplete {
       case Success(s) => onSuccess(s)
-      case Failure(e: CromwellFatalException) =>
+      case Failure(e: CromwellFatalException)=>
```

Comment:
Now it's taken longer to explain the emoji than just writing it out in full in the first place.